content/03.prior-knowledge.representation-and-transfer.md

```diff
@@ -1,55 +1,52 @@
-Despite the absence of AAV from the training set, MultiPLIER was able to learn a latent variable where the genes with the highest contributions encode antigens that the antineutrophil cytoplasmic antibodies (ANCA) form against in AAV and with higher expression in a group of samples from patients with AAV that were reported to have more severe disease [@doi:10.1002/art.27398].
-The utility of this approach stems from the fact that biological processes are often _shared_ between conditions–the same ANCA antigen genes are components of normal neutrophilic granule development that is likely captured or assayed in the collection of transcriptomic data used for training.
-MultiPLIER has additional attributes that make it practical for studying rare diseases: not all latent variables learned by a PLIER model are associated with input gene sets, and therefore may capture technical noise separately from biological signal, and we can use a single model to describe datasets from multiple tissues or cohorts that are not obviously directly comparable instead of attempting to reconcile results from multiple models (see _05.heterogeneity.md_).
-
-Taken together, the DeepProfile [@doi:10.1101/278739] and MultiPLIER [@doi:10.1016/j.cels.2019.04.003] results suggest transfer learning can be beneficial for studying rare diseases. 
+(Here "unseen" refers to the fact that these diseases were not in the training set.)
+PLIER is a matrix factorization approach that takes prior knowledge in the form of gene sets or pathways and gene expression data as input; some latent variables learned by the model will align with input gene sets  [@doi:10.1038/s41592-019-0456-1].
+We demonstrated that training on larger collections of randomly selected samples produced models that captured a larger proportion of input gene sets and better distinguished closely related signals, which suggests larger training sets produced models that are more suitable for biological discovery [@doi:10.1016/j.cels.2019.04.003].
+
+Though models trained on generic compendia had appealing properties, that alone does not guarantee suitability for describing rare diseases.
+We must examine the relevance of learned features to the disease under study.
+In Taroni et al., we found that the expression of latent variables that could be matched between the MultiPLIER model and a dataset-specific model were well-correlated, particularly when latent variables were associated with input gene sets [@doi:10.1016/j.cels.2019.04.003].
+Despite the absence of AAV from the training set, MultiPLIER was able to learn a latent variable where the genes with the highest contributions encode antigens that the antineutrophil cytoplasmic antibodies (ANCA) form against in AAV and with higher expression in more severe disease [@doi:10.1002/art.27398].
```

Comment:
I wonder if this sentence could be omitted? Might be overly detailed for this paper, not sure.
```suggestion
```

Reply:
Without this sentence, I think I would have to remove 

> –the same ANCA antigen genes are components of neutrophilic granule development that is likely captured or assayed in the collection of transcriptomic data used for training

In the next sentence. But the sentiment

> The utility of this approach stems from the fact that biological processes are often _shared_ between conditions

may be enough, thoughts?

Reply:
I'm going to plan leave this for now and see what #88 brings ✂️

Reply:
Sorry, just saw your follow up comment. I will consider this when I work on #88!